litmus-portal/backend/graphql-server/pkg/database/mongodb/project.go

```diff
@@ -0,0 +1,38 @@
+package database
+
+import (
+	"context"
+	"log"
+
+	"github.com/litmuschaos/litmus/litmus-portal/backend/graphql-server/graph/model"
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/mongo"
+)
+
+var projectCollection *mongo.Collection
+
+// CreateProject ...
+func CreateProject(ctx context.Context, project *model.Project) error {
```

Comment:
I can't find the mongodb struct for the users and project clusters, we cannot reuse the GQL struct because we need to add a `bson` tag for mongodb to specify the format in which the keys are stored.

Reply:
Yes for now I have used the same structs and their `bson` will be converting all the variables in lower cases. We can have this refactoring in the coming releases